src/main/kotlin/org/wycliffeassociates/otter/common/domain/content/WorkbookFileNamerBuilder.kt

```diff
@@ -0,0 +1,26 @@
+package org.wycliffeassociates.otter.common.domain.content
+
+import io.reactivex.schedulers.Schedulers
+import org.wycliffeassociates.otter.common.data.workbook.*
+
+object WorkbookFileNamerBuilder {
+    fun createFileNamer(
+        workbook: Workbook,
+        chapter: Chapter,
+        chunk: Chunk?,
+        recordable: Recordable,
+        rcSlug: String
+    ) = FileNamer(
+        bookSlug = workbook.target.slug,
+        languageSlug = workbook.targetLanguageSlug,
+        chapterCount = workbook.target.chapters.count().subscribeOn(Schedulers.io()).blockingGet(),
```

Comment:
not sure what I think of moving to an io thread and doing a blocking get especially if you're not wrapping the return value here in an observable and doing this async

Reply:
Yeah I think it's fine on the UI thread actually- it's a small operation that shouldn't cause any issues